README.md

```diff
@@ -337,5 +326,87 @@ Contact the dev team for the following:
 
 Once you have access to AWS LZ2 platform, login to the any of the environments to get access to the following: 
 
+- Project .env file can be found in the prod parameter store under pcc/.env
```

Comment:
The following can be found in AWS Systems Manager's Parameter Store on prod

- Project .env file under pcc/.env
- Database passwords under pcc/prod/postgres/password
- AWS Local Config - Service Account Tokens